java/bundles/org.eclipse.set.feature.table.pt1/src/org/eclipse/set/feature/table/pt1/sskz/SskzTransformator.java

```diff
@@ -123,8 +125,15 @@ private Table transform(final Iterable<Aussenelementansteuerung> controls,
 					.keySet().stream().map(this::fieldElementCase)
 					.toArray(Case[]::new);
 
-			fillSwitch(row, getColumn(cols, Betriebl_Bez_Feldelem), control,
-					fieldElementCases);
+			fillIterable(row, getColumn(cols, Betriebl_Bez_Feldelem), control,
+					(final Aussenelementansteuerung element) -> List
+							.of(fieldElementCases).stream()
+							.filter(c -> c.condition.apply(element))
+							.map(c -> c.filling.apply(element))
+							.flatMap(it -> StreamSupport
+									.stream(it.spliterator(), false))
+							.collect(Collectors.toList()),
```

Comment:
```suggestion
							.toList(),
```
From Stream can direct `toList()` 😄

Reply:
@TruongQuangSB I changedit